DiscoveryServiceSpecification/V3.1_SSP-001.yaml

```diff
@@ -0,0 +1,174 @@
+openapi: 3.0.3
+info:
+  title: DotAAS Part 2 | HTTP/REST | Discovery Service Specification (V3.1 Draft, not released)
+  description: >-
+    The entire Full Profile of the Discovery Service Specification as part of the [Specification of the Asset Administration Shell: Part 2](http://industrialdigitaltwin.org/en/content-hub).  
+    
+    Publisher: Industrial Digital Twin Association (IDTA) April 2023
+  contact:
+    name: Industrial Digital Twin Association (IDTA)
+    email: info@idtwin.org
+  license: 
+    name: CC BY 4.0
+    url: https://creativecommons.org/licenses/by/4.0/
+  version: V3.1.0_SSP-001
+  x-profile-identifier: https://admin-shell.io/aas/API/3/1/DiscoveryServiceSpecification/SSP-001
+servers:
+- url: 'https://admin-shell.io/api/v3.1'
+- url: 'https://example.com/'
```

Comment:
why two url?

Reply:
To make clear that the "/api/v3.1" path segments are not mandatory.